### ISSUE TITLE
administration/reset password does not work in arch1900

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#1010]
gives a general error toast and does not set a temporary password.

**Comment by urapadmin** [comment]
### @test kiosk 0.11
it did not work in any project. Now it does again. Slight improvement: The message with the new temporary password does not vanish after 5 seconds anymore.

**Comment by lbestock** [comment]
Worked to reset. New temp password worked to login (the extra time to remember the temp password helped). But when trying to reset my password, there was an autofill password already there. I selected it all to replace it, and when I did that I was kicked back to the login page and given the message below. This happened twice, and the temp login continued to work. When I kept the autofill (which was my old password) and then repeated my password it worked and I could reset my password, but this shouldn't happen.
![image](https://user-images.githubusercontent.com/49291909/115129441-debfb480-9fb3-11eb-931a-5173fd57a662.png)
